cmd/rpcdaemon/cli/config.go

```diff
@@ -53,7 +54,7 @@ var rootCmd = &cobra.Command{
 }
 
 const JwtTokenExpiry = 5 * time.Second
-const JwtDefaultFile = "jwt.hex"
+const JwtDefaultFile = "/tmp/jwt.hex"
```

Comment:
Should use `os.TempDir()` instead of `/tmp`, at least in order to be compatible with Windows.

Reply:
Actually, I think it's better to follow geth's example and place this file inside datadir.

Reply:
what if we do not specify any datadir in rpcdaemon, or what if we run the daemon on another machine than erigon?

Reply:
I suggest to use current directory

Reply:
OK, let's stick with the current directory.